backend/uclapi/roombookings/management/commands/update_gencache.py

```diff
@@ -35,53 +36,70 @@ def handle(self, *args, **options):
 
         self.stdout.write("Selecting a clone table...")
         lock = Lock.objects.all()[0]
-        curr = BookingA if lock.bookingA else BookingB
+        current_bucket = BookingA if lock.bookingA else BookingB
```

Comment:
I applaud this name change 👏 I always got confused with what `curr` meant

Reply:
Yeah I think we were due a bit of a refactor of this function :P